tools/travis/build-helm.sh

```diff
@@ -56,14 +56,15 @@ statefulsetHealthCheck () {
       break
     fi
 
-    kubectl get pods --all-namespaces -o wide --show-all
+    kubectl get pods -n openwhisk -o wide
 
     let TIMEOUT=TIMEOUT+1
     sleep 10
   done
 
-  if [ ! $PASSED ]; then
+  if [ "$PASSED" == "false" ]; then
     echo "Failed to finish deploying $1"
+    kubectl get pods --all-namespaces -o wide
 
```

Comment:
Do you want to use `-n openwhisk` here?

Reply:
No.  If something goes wrong I want to log the state of all the pods.  Sometimes the problem is a pod in the kube-system namespace is in a crash loop.  Added a comment to the script (here and in the other two places).